frontend/app/tests/unit/store/settings/actions.spec.ts

```diff
@@ -13,7 +13,8 @@ import {
   THOUSAND_SEPARATOR,
   DECIMAL_SEPARATOR,
   CURRENCY_LOCATION,
-  REFRESH_PERIOD
+  REFRESH_PERIOD,
+  EXPLORERS
```

Comment:
so this is also stored in the frontend settings of the user DB, right?

Reply:
Exactly.